### ISSUE TITLE
The component has no name, add 'name' property on the Vue component

### ISSUE THREAD

**Issue by KnaanBraz** [vue-styleguidist/vue-docgen-api#30]
```
 The component has no name, add 'name' property on the Vue component 
```

My component has a parameter for the name property...

**Comment by KnaanBraz** [comment]
```js
const NAME = 'component-name'

export default {
  name: NAME
}
```

**Comment by adi518** [comment]
There's no fallback for `parse`. I think if it cannot parse the file, it should require it and then try to pick the `name` property. Anyhow, I don't think it should throw an error in most cases, because then it's not possible to obtain whatever other data it did manage to pick.

**Comment by elevatebart** [comment]
Hello @KnaanBraz trying to cover our bases  here. do you still meet tis issue with the current version ?
